charts/nri-bundle/Chart.yaml

```diff
@@ -16,7 +16,7 @@ sources:
   - https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-pixie
   - https://github.com/newrelic/newrelic-infra-operator/tree/master/charts/newrelic-infra-operator
 
-version: 5.0.2
+version: 5.1.0
```

Comment:
What is going to be the new default KSM? If newest KSM is the default, I think we should bump the major.

Reply:
I think the newest KSM is the default. I will bump the major.